source/docs/3/origin_destination_matrix.rst

```diff
@@ -90,115 +94,57 @@ Procedure
 
   .. image:: /static/3/origin_destination_matrix/images/13.png
     :align: center
   
-14. Enter the following query and click :guilabel:`Execute`. The results will be displayed in the panel below. As expected, we have 9 rows in the result - the shortest path destination for each origin point. Check and select :guilabel:`Column with unique values` as ``origin_id``. Enter ``nearest_destinations`` as the :guilabel:`Layer name (prefix)`. Click :guilabel:`Load`.
+14. Enter the following query in :guilabel:`SQL query` dialog box. Enter ``geometry`` as the :guilabel:`Geometry field` and, select ``LineString`` as the :guilabel:`Geometry type`. Click :guilabel:`Run`.
 
   .. code-block:: none
 
     select origin_id, destination_id, min(total_cost) as shortest_distance 
-    from 'Output OD Matrix' group by origin_id
+    from input1 group by origin_id
 
   .. image:: /static/3/origin_destination_matrix/images/14.png
     :align: center
   
-15. A new virtual layer ``nearest_destinations`` will be added to the :guilabel:`Layers` panel. This table has the result of our analysis. Nearest mental health center for each of the 9 origin points. Let's try a few different ways to visualize and validate these results. Go to :menuselection:`Processing --> Toolbox`. Search for and locate the :menuselection:`Join attributes by field value` algorithm. Double-click to launch it.
+15. A new virtual layer ``SQL Output`` will be added to the :guilabel:`Layers` panel. This Layer has the result of our analysis. Nearest Adult mental health center for each of the 1000 origin points. Let's try a few different ways to visualize and validate these results. 
 
   .. image:: /static/3/origin_destination_matrix/images/15.png
     :align: center
   
-16. Select ``origin_points`` as the :guilabel:`Input layer` and ``OBJECTID`` as the :guilabel:`Table field`. Set ``nearest_destinations`` as the :guilabel:`Input layer 2` and ``origin_id`` as the :guilabel:`Table field 2`. Click the :guilabel:`...` button next to :guilabel:`Layer 2 fields to copy` and select ``destination_id`` and ``shortest_distance``. Click :guilabel:`Run`.
+16. Note that even though the lines connecting the origin and destination is a straight-line, the destination was found using the distance along with the network. It will be much useful visualization to show the actual shortest-path between each origin-destination. As of now, there is no easy way to generate the shortest-path between multiple origin-destination pairs the way we generated the distance matrix. But I will demonstrate a way to use some python scripting to generate this visualization. First, let's run the shortest path algorithm on 1 pair. Locate the :menuselection:`QNEAT3 --> Routing --> Shortest path (point to point)` algorithm and launch it.
```

Comment:
This text from this step onwards doesn't make sense at all. There are important validation steps missing. Please refer to chat screenshot. Remove reference to python scripting. Say we will validate the results using 1 pair.

Reply:
Updated.